platforms/android/build_sdk.py

```diff
@@ -260,6 +260,9 @@ def build_library(self, abi, do_install):
         if do_install:
             cmd.extend(["-DBUILD_TESTS=ON", "-DINSTALL_TESTS=ON"])
 
+        if use_media_ndk:
+            cmake_vars['WITH_ANDROID_MEDIANDK'] = "ON"
```

Comment:
Currently `WITH_ANDROID_MEDIANDK` is enabled by default for `ANDROID_NATIVE_API_LEVEL`>20. So we should add `--no-media-ndk` option to be able to disable it.

Reply:
Done.